bigbluebutton-html5/imports/startup/client/base.jsx

```diff
@@ -114,6 +114,8 @@ class Base extends Component {
           userJoinPushAlerts,
         } = Settings.application;
 
+        if (!userJoinAudioAlerts && !userJoinPushAlerts) return;
```

Comment:
If you place this line before `const users = ` we don't even need to observe in the majority of the cases

Reply:
I think this is still needed because we are tracking additions (`.observe` function) in user's collection. If the settings is changed during the session we need to test it before showing or not the notification.